packages/venia-ui/lib/drivers/index.js

```diff
@@ -55,8 +54,8 @@ export { connect } from 'react-redux';
  * components by configuring the module resolver in their build system.
  *
  * Because this is all one file, a third party can use a Venia component that
- * contains <Query> and <Link> elements, even without an Apollo client or
- * a React Router in context, simply by adding this Webpack config:
+ * contains <Link> elements even without a React Router in context,
```

Comment:
Just because we're using hooks doesn't mean an Apollo client is no longer in context. We're still rendering `ApolloProvider`, and `useQuery` is still consuming Apollo context.